hydroflow/examples/two_pc/coordinator.rs

```diff
@@ -41,24 +41,38 @@ pub(crate) async fn run_coordinator(opts: Opts, subordinates: Vec<String>) {
         hf.make_edge::<_, VecHandoff<String>, _>("subordinates_out_tee_2");
     let (subordinates_out_tee_3_push, subordinates_out_tee_3_pull) =
         hf.make_edge::<_, VecHandoff<String>, _>("subordinates_out_tee_3");
+    let (subordinates_out_tee_4_push, subordinates_out_tee_4_pull) =
+        hf.make_edge::<_, VecHandoff<String>, _>("subordinates_out_tee_4");
 
     // create a flow to populate the subordinate handoffs
-    hf.add_subgraph(
```

Comment:
we need some syntax sugar for this, or to revisit `TeeingHandoff`, pretty gross you have to manually binarily tee like this